index.html

```diff
@@ -0,0 +1,227 @@
+      <div class="speaker-page-div">
+      </div>
+        <!--    add here with js   -->
+    </section>
+    
+    <!--    about me page   -->
+
+    <section class="about-page" id="about-page">
+
+      <!--    titles    -->
+
+      <div class="about-page-tittles">
+        <h2 class="about-page-h2">"Security is the future"</h2>
+        <h1 class="about-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+      </div>
+
+      <!--    description of the event    -->
+
+      <div class="about-page-text">
+        <p class="about-page-text-p">Blockchain Summit is back, 
+          one of the most popular and important crypto events worldwide,
+           aimed at market leaders, Web3 sector entrepreneurs, investors,
+            and developers. To attend with a 15% discount,
+             use the promotional code BCS2023.
+        </p>
+      </div>
+      
+      <!--    contact info    -->
+
+      <div class="about-page-info">
+        <h2 class="about-page-info-date">Please contact us per Email for nay further questions about Block-chain first TECH SUMMIT 2023:</h2>
+        <h3 class="about-page-info-place">mail: cryptoconferences@gmail.com & instagram: cryptOk</h3>
+      </div>
+      
+    </section>
+
+    <section class="about-page-past-content">
+      <!--  titles of page past  -->
+      <div class="about-page-past">
+          <h1 class="about-page-past-h1">Creative Commons Block-chain TECH SUMMIT 2023 Logo</h1>
+          <div class="about-page-past-border"></div>
+          <h2 class="about-page-past-h2">The logo Block-chain TECH SUMMIT 2023 was decided through 
+            the logo competition from 8, June to 7, July.</h2>
+      </div>
+      <!--  my logo-->  
+      <div class="about-page-past-logo">
+        <img src="image/logo-no-circle.png" alt="" width="100px">
+        <h1 class="about-page-past-logo-text">Libra</h1>
+      </div>
+      <!--    --> 
+      <div class="about-page-past">
+        <h1 class="about-page-past-h1">See the past Block-chain TECH SUMMIT 2023</h1>
+        <div class="about-page-past-border"></div>
+        <h2 class="about-page-past-h2">Take a look at the past two Block-chain TECH SUMMIT 
+          which took place in Buenos Aires and in Warsaw.</h2>
+      </div>
+      <!--  img of page past  -->
+      <div class="about-page-past-img">
+        <!-- agregar background img -->
+        <div class="about-page-past-background-1">
+         <h1 class="about-page-past-media-h1-1">2018</h1>
+         <h2 class="about-page-past-media-h2-1">First Congress in Bs As</h2>
+        </div>
+        <!-- agregar background img -->
+        <div class="about-page-past-background-2">
+         <h1 class="about-page-past-media-h1-1">2020</h1>
+         <h2 class="about-page-past-media-h2-2">Second Congres in Germany</h2>
+        </div>
+      </div>
+    </section>
+    <section class="sponsor-page">
+      <div class="sponsor-page-past-sponsors">
+        <div><h1>Our Partners</h1></div>
```

Comment:
- [ ] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍